doc/command_ref.rst

```diff
@@ -457,7 +457,9 @@ Alias Examples
     Adds new alias command called "rm" which does the same thing as the command "remove".
 
 ``dnf alias add update="\update --skip-broken --disableexcludes=all --obsoletes"``
-    Adds new alias command called "update" which does the same thing as the command "update", but with options ``--skip-broken --disableexcludes=all --obsoletes``.
+    Adds new alias command called "update" which does the same thing as the command "update",
+    but with options ``--skip-broken --disableexcludes=all --obsoletes``. Note that the
+    original command "update" is prefixed with "\\\\" to prevent infinite alias processing.
```

Comment:
I'd phrase it like this:
> Adds **a** new **command alias** called **`update`,** which **works the same** as the `update` command, **with additional options**. Note that the original **`update` command** is prefixed with **a "\\\\"** to prevent **a loop in alias processing**.

- saying "command alias" is the right way IMO, instead of "alias command"
- use backticks to highlight the command names
- I think you should use two backslashes in the code, which should result in a single backslash in the resulting text, and single backslash is what's used in the command

(feel free to fix the other example in similar fashion if you want :slightly_smiling_face:)

Reply:
Thanks for review, fixed (except I believe it should be a single backslash in the code - it's even used in the documentation before).

Reply:
Right, a single backslash works. Thanks, merging.